v3/src/v2/codap-v2-types.ts

```diff
@@ -97,12 +97,14 @@ export interface ICodapV2PlotModel {
 }
 
 export interface ICodapV2GraphStorage {
-  _links_: {
+  _links_: Record<string, {
```

Comment:
```suggestion
  _links_: {
```
This use of `Record<>` implies that every property of `_links_` has the form of the entire structure. This should be put back the way it was.